modules/dqt/php/provisioners/categoriesinstance.class.inc

```diff
@@ -0,0 +1,15 @@
+<?php
```

Comment:
I don't know where to put those classes. There will be many. 
Shold I try to use the DataDictionnary categories instead?

Reply:
I think it makes sense to use the Data Dictionary categories.. what's the point in creating a new parallel type?